src/octoprint/templates/snippets/settings/webcam/webcamStreamWebrtcIce.jinja2

```diff
@@ -0,0 +1,7 @@
+<div class="control-group" title="{{ _('Endpoint to use for WebRTC ICE')|edq }}">
+    <label class="control-label" for="settings-webcamStreamWebrtcIce">{{ _('WebRTC ICE endpoint') }}</label>
+    <div class="controls">
+        <input type="text" class="input-block-level" data-bind="value: webcam_streamWebrtcIce, valueUpdate: 'afterkeydown'" id="settings-webcamStreamWebrtcIce">
+        <span class="help-block"><span class="label">{{ _('WebRTC') }}</span> {% trans %}Only used for WebRTC streams. Provide the endpoint to use for WebRTC's Interactive Connectivity Establishment (ICE). Usually a STUN server. If unsure, leave at the default of <code>stun:stun.l.google.com:19302</code>.{% endtrans %}</span>
```

Comment:
This can be a `STUN` or `TURN` server. [Docs](https://developer.mozilla.org/en-US/docs/Web/API/RTCConfiguration/iceServers)

`Only used for WebRTC streams. Provide the ICE or STUN server to use for WebRTC's Interactive Connectivity Establishment (ICE). If unsure, leave the default of <code>stun:stun.l.google.com:19302</code>.`

Reply:
The short explanation is that a STUN server just helps negotiate NAT traversal for UDP while a TURN server relays all the traffic. Either can be used to get around NATs, but TURN servers aren't free because of the bandwidth usage.